specification/servicefabric/resource-manager/readme.md

```diff
@@ -26,17 +26,17 @@ These are the global settings for the ServiceFabricClient API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2016-09
+tag: package-2017-07
 ```
 
 
-### Tag: package-2016-09
+### Tag: package-2017-07

```

Comment:
Thank you for adding a readme file. Please, add a new tag and keep `package-2016-09` untouched. This way, we can generate SDK for different versions on demand and keep track of how we did it before.

Reply:
Done

Reply:
@juhacket thank you, I will keep posting mini-review to fix CI until ARM approval. After that, I will send full review.